dataAccessObjects.py

```diff
@@ -107,6 +107,23 @@ def getIdData(self,Id):
             #TODO raise exception
             return None
 
+    def insertData(self,data):
+        sql = 'INSERT INTO {tableName} ' \
+            + '({dataName}_id, type_id, {dataName}_system_name, ' \
+            + '{column}, {dataName}_start_date, ' \
+            + '{dataName}_end_date, {dataName}_start_time, {dataName}_end_time, ' \
+            + '{dataName}_display_time, user_id)' \
+            + ' VALUES ' \
+            + '("{data[id]}", {data[typeId]}, "{data[systemName]}", {value}, "{data[startDate]}",' \
+            + ' "{data[endDate]}", "{data[startTime]}", "{data[endTime]}", {data[displayTime]}, {data[userId]})'
+        if self.tableName == "image_data":
+            sql = sql.format(column="img_thumbnail_name, img_file_name",value='"{data[thumbnailName]}", "{data[fileName]}"')
+        elif self.tableName == "text_data":
+            sql = sql.format(column="text_invisible_title",value='{data[invisibleTitle]}')
+        
```

Comment:
line 122 `sql = sql.format(column="text_invisible_title",value='{data[invisibleTitle]}')`
should be
`sql = sql.format(column="text_invisible_title",value='"{data[invisibleTitle]}"')`
because text_invisible_title is string type.